src/components/Context/injectEnvironment.js

```diff
@@ -10,18 +10,28 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { deepAssign } from "../../utils";
+import { deepAssign, isNumber } from "../../utils";
 
 export default window => {
   return xdm => {
-    const { innerWidth, innerHeight } = window;
+    const {
+      document: { documentElement: { clientWidth, clientHeight } = {} }
```

Comment:
So AppMeasurement is using offsetWidth here. What made you pick clientWidth? There seems to be a little bit of a difference in that offsetWidth includes some additional browser elements etc.

Reply:
I was just following the stack overflow answer linked above. I didn't realize appMeasurement was using documentElement as a fallback. offsetWidth and offsetHeight are probably better.

Reply:
Using the property that will fluctuate the least (based on margin settings etc.) to enable for common resolutions to group well would probably be preferred.